NEWS

```diff
@@ -33,6 +33,18 @@ NEWS - user visible changes				-*- outline -*-
    patterns where invalid numerical data (e.g, SPACES) encode "absent"
    data
 
+** #NNN: dialects without support of single byte binary data
+   will now generate at least 2 bytes of storage for COMP-5 elements; this
+   applies to all non-standard dialects but GnuCOBOL (default) and Micro Focus;
+   that changes the group length of records containing elements with less than
+   3 digits and also passes different sizes via CALL, which may need program
+   adjustments.
+
+** -std=acu / -std=acu-strict now generate BINARY and COMP-5 with at least
+   2 bytes of storage; the comment for #NNN above applies; if you want
```

Comment:
Ah, of course I forget something...

as we do change something that may break on recompilation, we definitely need an upstream bug report for this outlining the issue, fix and possible side effects and in these changes (Changelog + NEWS) replace NNN with the new bug number.